### ISSUE TITLE
travis fails to build ldapadmin on master

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/georchestra#1394]
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:0.0.27:npm (npm install) on project ldapadmin: Failed to run task: 'npm install --color=false' failed. (error code 1) -> [Help 1]

https://travis-ci.org/georchestra/georchestra/builds/135818601

**Comment by fvanderbiest** [comment]
Fixed (eg https://travis-ci.org/georchestra/georchestra/builds/162134768)

**Comment by fvanderbiest** [comment]
Not fixed ...
https://travis-ci.org/georchestra/georchestra/builds/165473316

[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:0.0.27:npm (npm install) on project ldapadmin: Failed to run task: 'npm install --color=false' failed. (error code 1) -> [Help 1]

**Comment by pmauduit** [comment]
Will it be relevant to try a more recent version of the frontend maven plugin ? http://mvnrepository.com/artifact/com.github.eirslett/frontend-maven-plugin/1.0

**Comment by tonio** [comment]
Yes, I don’t know which version of npm is installed, but some insight [here](https://github.com/npm/registry/issues/10#issuecomment-217141066)…

**Comment by pmauduit** [comment]
WIP onto https://github.com/pmauduit/georchestra/tree/travis-ldapadmin-1394 (just bumping versions of maven-plugin, nodejs, npm, 2 builds successful so far on travis)

**Comment by fvanderbiest** [comment]
This was merged into master several days ago, but I'm still exprimenting the same error while building master :-(

**Comment by fvanderbiest** [comment]
I fear this is another issue here...
```
[ERROR] npm ERR! Command failed: git clone --template=/home/deploy/.npm/_git-remotes/_templates --mirror git://github.com/btford/route-recognizer.git /home/deploy/.npm/_git-remo
tes/git-github-com-btford-route-recognizer-git-99ace589
[ERROR] npm ERR! fatal: unable to connect to github.com:
[ERROR] npm ERR! github.com[0: 192.30.253.112]: errno=Connection timed out
```
(git protocol being blocked by proxies)

**Comment by pmauduit** [comment]
Then forget about Travis ;-)

**Comment by fjacon** [comment]
à fermer ?
